src/Makefile

```diff
@@ -277,7 +277,7 @@ mpi-stubs:
 sinclude ../lib/python/Makefile.lammps
 install-python:
 	@$(PYTHON) ../python/install.py -v ../src/version.h \
-		-m ../python/lammps.py -l ../src/liblammps.so
+		-m ../python/lammps.py -l ../src/liblammps.*[dylib,so]
```

Comment:
I guess you want `liblammps.*{dylib,so}`.

Reply:
That is also matching - `liblammps.0.dylib`